modules/ROOT/pages/slsa.adoc

```diff
@@ -0,0 +1,130 @@
+verification process. For RHTAP the same key is used to sign the attestation and the image itself. As mentioned above, there is experimental
+support for verifying keylessly signed attestations.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_build_scripted_build__subject_build_task_matches[Provenance subject matches build task image result]
+a| SLSA recommends an artifact verifier checks that the that provenance statement’s subject matches the digest of the artifact in question.
+This is a good sanity check to confirm that the provenance being considered is for expected image. Because EC uses Cosign under the hood, and
+Cosign does this check as part of its signature verification, we get this confirmed automatically. The EC rule linked here does an additional
+sanity check by comparing the image reference produced by the build task and confirming that also matches the provenance subject.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_provenance_available__attestation_predicate_type_accepted[Expected attestation predicate type found]
+a| SLSA suggests that the predicate type in the artifact's provenance should be `https://slsa.dev/provenance/v1`. This rule confirms that requirement,
+and will fail if the predicate type found to be any other value.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#provenance_materials__git_clone_source_matches_provenance[Git clone source matches materials provenance]
+a| This is a good sanity check related to artifact verification that isn't specifically mentioned in the SLSA guidelines. In RHTAP build pipelines the
+git clone task provides information about the git repo and the commit sha used. This rule confirms that the url and the commit sha from the git clone
+task match what appears in the "materials" provenance.
+
+|===
+
+== Notes and caveats
+
+Because EC was designed initially for use with RHTAP, and RHTAP uses Tekton
+CI/CD, many of the specific rules mentioned above are currently Tekton specific
+in that they use details about the Tekton pipeline run, which are currently
+made available in the SLSA attestation created by Tekton Chains.
+
+This is not a limitation of EC itself, but rather just an attribute of the
+xref:ec-policies:ROOT:release_policy.adoc[policies created for RHTAP] in their
+current state. In future we're aiming to extract the Tekton specific rules from
+the general SLSA rules to make the rules more generally useful. It's expected
+that policy rules for build systems not based on Tekton and Tekton Chains could
+be created easily.
+
+Note also that since EC can conveniently apply any policy defined in OPA/Rego,
```

Comment:
Would it be worth embedding links for OPA or Rego to take them somewhere that describes those things?